topics/go/algorithms/slices/max/max.go

```diff
@@ -1,24 +1,21 @@
 package slices
 
-// Max returns the maximum integer in the slice.
-func Max(n []int) int {
+import "fmt"
 
-	// If the length of the slice is 1 then return the integer at index 0.
-	if len(n) == 1 {
-		return n[0]
+// Max returns the maximum integer in the slice.
+func Max(n []int) (max int, err error) {
```

Comment:
Don't name the return arguments.

Reply:
absolutely.  i typically don't but i was trying to save a line or two.  i figured i name max in the function declaration instead of dedicating a line to it.